index.js

```diff
@@ -123,17 +134,17 @@ export class SparqlHandler {
     } else if (!exists) {
       return next()
     } else {
-      const query = isContainer ? this.buildContainerGraphQuery(iri) : this.buildResourceGraphQuery(iri)
+      const query = isContainer
+        ? this.buildContainerGraphQuery(iri)
+        : this.buildResourceGraphQuery(iri)
 
-      const { status, headers, stream } = await this.graphStream(iri, query, req.headers.accept)
+      const { status, stream } = await this.graphStream(iri, query,
+        req.headers.accept)
 
       if (!stream) {
         return next()
       }
       res.status(status)
-      Object.keys(headers).forEach(name => {
-        res.setHeader(name, headers[name])
-      })
```

Comment:
I don't think you can remove this. The SPARQL endpoint sends the `content-type`, and maybe some other headers that must be forwarded.